eclipse/eclipse-2018-12/gcp-eclipse-2018-12.tpd

```diff
@@ -0,0 +1,52 @@
+/*
+ * Target Platform Definition created using Eclipse TPD editor
+ * (http://download.eclipse.org/cbi/tpd/3.0.0-SNAPSHOT/).
+ *
+ * If you make changes to this file, either:
+ *
+ *    * Right-click in the editor and choose 'Create Target Definition File'
+ *      to update the corresponding .target file.
+ *    * Right-click in the editor and choose 'Set as Target Platform'
+ *      to update your IDE's target platform (regenerates the .target too)
+ */
+target "GCP for Eclipse 2018-12" with source requirements
+// Eclipse Platform 4.10
+// Eclipse WTP 3.12
+
+// Eclipse 2018-12 RC2 (composite at http://download.eclipse.org/releases/2018-12/)
+location "http://download.eclipse.org/staging/2018-12/" {
```

Comment:
"release" instead of "staging"? (Maybe not.)

Reply:
I hesitated in putting `/releases` as the composite repository there is disabled for a couple of days before the official release.  But I think the actual repositories it points to are still available.  And `/staging` will disappear after the official release.  So I'll put in the full path the latest RC for 2018-12.